suite/tests/CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ endif (UNIX)
 set(main_run_list
   # our main configuration
   "SHORT::-code_api"
+  # no_intercept_all_signals tests.
+  "SHORT::ONLY::signal_intercept$::-no_code_api -no_intercept_all_signals"
```

Comment:
Put a #5261 comment about removing this option

Reply:
Done.